Framework/src/ObjectsManager.cxx

```diff
@@ -70,6 +70,9 @@ void ObjectsManager::startPublishing(TObject* object, PublicationPolicy publicat
     ILOG(Warning, Support) << "Object is already being published (" << object->GetName() << "), will remove it and add the new one" << ENDM;
     stopPublishing(object->GetName());
   }
+  if (mergers::isMergeable(object)) {
```

Comment:
I think this should be also disabled by the presence of `QUALITYCONTROL_POSTPROCESSINTERFACE_H` and `IgnoreMergable==true`, otherwise we will see a lot of spurious warnings.

Reply:
@justonedev1 can you explain how your most recent changes make `QUALITYCONTROL_POSTPROCESSINTERFACE_H` disable this warning? I'm convinced they do.

Reply:
They do not. Somehow my changes got deleted, sorry about that

Reply:
it should be done now